analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CommandPath.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security;
+
+public class Program
+{
+    private string compliantField = @"C:\file.exe";
+    private string noncompliantField = "file.exe";
+
+    private Process field = Process.Start("file.exe");                      // Noncompliant
+    public Process PropertyRW { get; set; } = Process.Start("file.exe");    // Noncompliant
```

Comment:
maybe a property that doesn't have anything assigned would fix the partial coverage problem?

Reply:
Not sure, let's try

Reply:
Did not help :(